store/store.go

```diff
@@ -196,29 +195,6 @@ func (s *DefaultStore) GetSignature(ctx context.Context, height uint64) (*types.
 	return &signature, nil
 }
 
-// SaveExtendedCommit saves extended commit information in Store.
-func (s *DefaultStore) SaveExtendedCommit(ctx context.Context, height uint64, commit *abci.ExtendedCommitInfo) error {
```

Comment:
while this is tied to abci i think we will want something like this in order to facilitate the attester system and the oracle system. I would disagree with this pr as the design is not thought through all the way